### PR TITLE
add foundation-emails v2.0.0 w/ npm auto-update

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #7360 

@PeterDaveHello please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/zurb/foundation-emails
 * Official website (optional, not the repository): http://foundation.zurb.com/emails.html
 * NPM package url (optional): https://www.npmjs.com/package/foundation-emails
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 3574
   - Count of watchers: 170
   - Count of forks: 2570
 * NPM download stats (optional):
   - Downloads in the last day: 185
   - Downloads in the last week: 983
   - Downloads in the last month: 3333

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@kennynaoh Pleasr ask other @cdnjs/intern2 members first.

**Comment by PeterDaveHello** [comment]
Why did I see d6be1ad for grommet here?

**Comment by kennynaoh** [comment]
I have modified this.
Thanks

**Comment by PeterDaveHello** [comment]
@kennynaoh Call your reviewer first.

**Comment by kennynaoh** [comment]
@pvnr0082t please help me check this PR
thanks

File: ajax/libs/foundation-emails/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/zurb/foundation-emails.git"
+  },
+  "license": "MIT",
+  "eyeglass": {
+    "name": "foundation-emails",
+    "sassDir": "scss",
+    "needs": "^0.8.0"
+  },
```
Comment (pvnr0082t) [inline ajax/libs/foundation-emails/package.json]: @PeterDaveHello Is that ```eyeglass``` should be removed? I am not sure for this.

Comment (PeterDaveHello) [inline ajax/libs/foundation-emails/package.json]: Yes, should be removed.

**Comment by kennynaoh** [comment]
@pvnr0082t 
Help me check again
Thanks

File: ajax/libs/foundation-emails/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "foundation-emails",
+  "filename": "foundation.min.css",
+  "npmName": "foundation-emails",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "foundation*"
```
Comment (pvnr0082t) [inline ajax/libs/foundation-emails/package.json]: @kennynaoh Should be ```**/*```.

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have updated.
Please check again.
Thanks

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this pull request is ok. Please help me check it again, thank you.

File: ajax/libs/foundation-emails/package.json (hunk 2 above)
Comment (PeterDaveHello) [inline ajax/libs/foundation-emails/package.json]: Nice catch.

**Comment by PeterDaveHello** [comment]
1. Bad PR title.
2. Please always rebase your branch with the latest master branch once you modified your PR.

**Pull request merged**